pkgs/development/libraries/science/math/clblas/cuda/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, gfortran
+, blas
+, boost
+, python
```

Comment:
you can remove `python` here and from the `buildInputs` as it is already provided with `buildPythonPackage`.